tests/kernel/blktests.pm

```diff
@@ -0,0 +1,78 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018-2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Block device layer tests
+# Maintainer: Michael Moese <mmoese@suse.de>, Sebastian Chlad <schlad@suse.de>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+#use utils 'zypper_call';
+#use power_action_utils 'power_action';
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    #below variable exposes blktests options to the openQA testsuite
+    #definition, so that it allows flexible ways of re-runing the tests
+    my $quick   = get_required_var('blk_QUICK');
+    my $exclude = get_required_var('blk_EXCLUDE');
+    my $config  = get_required_var('blk_CONFIG');
+    my $tests   = get_required_var('blk_TESTS');
+    my $device  = get_required_var('blk_DEVICE_ONLY');
+
+    script_run('zypper -n in git-core make gcc coreutils awk util-linux fio gcc-c++',       180);
+    script_run('git -c http.sslVerify=false clone https://github.com/osandov/blktests.git', 120);
```

Comment:
I suggest by default to always use `assert_script_run` to abort early on failures to not get confused by scripts that continue in case e.g. the complete network is down. Especially important in the next call when you do "cd". Could be dangerous to work in the wrong directory when the "cd" never succeeded :D

Reply:
ah this makes sense actually. I guess overall here the confusion would be fairly limited as the openqa code is super simple, but I see your point. Thx. I will change some lines in there then to use assert_script_run